cde-core/resource/js/cdf-dd.js

```diff
@@ -1881,11 +1881,12 @@ $(function() {
     },
     content: function(data, settings, original) {
       // If it is string assume it is an array.
+      var json;
       if(String == data.constructor) {
-        eval('var json = ' + data);
+        json = JSON.parse(data);
```

Comment:
I'm getting an error here when using trying to add a parameter as a listener to a component, via the cde editor.
Looking into it a bit, the problem seems to be the json string is malformed (uses ' as instead of " as property/value delimiter ).
For this particular error, reworking the json String builder for the ListenersRenderer to use double  quotes instead of single should do the trick (https://github.com/webdetails/cde/blob/master/cde-core/resource/js/cdf-dd-components-selectors.js#L30), I'm currently looking for other usages of jeditable